plugins/tokenVoting/components/proposal/header.tsx

```diff
@@ -125,23 +128,45 @@ const ProposalHeader: React.FC<ProposalHeaderProps> = ({
   );
 };
 
-const getProposalStatusVariant = (proposal: Proposal) => {
-  return {
-    variant: proposal?.active
-      ? "info"
-      : proposal?.executed
-      ? "primary"
-      : proposal?.tally?.no >= proposal?.tally?.yes
-      ? "critical"
-      : ("success" as AlertVariant),
-    label: proposal?.active
-      ? "Active"
-      : proposal?.executed
-      ? "Executed"
-      : proposal?.tally?.no >= proposal?.tally?.yes
-      ? "Defeated"
-      : "Executable",
-  };
+const getProposalStatusVariant = (proposal: Proposal, tokenSupply: bigint) => {
```

Comment:
Shouldn't this be added to the proposal component in the proposal list?
This function could be in the helpers or something where both components (header and proposal card) can use it